pyvista/plotting/widgets.py

```diff
@@ -1301,6 +1303,21 @@ def add_mesh_threshold(
             to intersect the threshold bound, rather than the set of
             discrete scalar values from the vertices.
 
+        all_scalars : bool, optional
```

Comment:
```suggestion
        all_scalars : bool, default: False
```

Reply:
What's the preferred way to do this? IMO, bool, optional makes sense (being that default False is implied). This is the same for empty strings. But what's more "proper"? I think we do both throughout the codebase

Reply:
We wrote in the contributing section to use `optional` when the default is `None` to align  with `typing.Optional`, i.e. it doesn't match the type given for three partner, and `default: value`, when there is a default value that should match the type.

Reply:
Ah I missed that. I may need to follow up and correct this in other places